news/2020-07-01-new-featured-artist-leaf-xceed.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: LEAF XCEED Music Division"
+date: 2020-07-01 08:00:00 +0000
+---
+
+We're proud to welcome **LEAF XCEED Music Division** aboard as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/88/header.jpg)
+
+A blast from the past coming straight at you in 2020, [**LEAF XCEED Music Division**](https://osu.ppy.sh/beatmaps/artists/88) is here to stay as our newest addition to the Featured Artist roster. 
+
+Most well known in osu! from his tracks *Fur Elisa* ([mapped](https://osu.ppy.sh/beatmapsets/23916) by [Louis Cyphre](https://osu.ppy.sh/users/186243)), *YuYu Metal* ([mapped](https://osu.ppy.sh/beatmapsets/45528) by [DoKito](https://osu.ppy.sh/users/537084)), and more recently *Guitar vs Piano* ([mapped](https://osu.ppy.sh/beatmapsets/518003) by [Shiro](https://osu.ppy.sh/users/113005)), we're excited to announce a new ensemble of 8 tracks including these timeless classics are now available, all bundled up in a neat album rather aptly titled *osu! 「SELECT」*.
+
+In addition, the Mappers' Guild has been hard at work creating new takes on these classic metal hits to quite literally, bring your experience into the modern era. Don't discount the older tracks though - they're every bit as good now as they were back in 2011!
+
+Check out a few previews below for yourself and see what all the hype is about:
+
+### Fur Elisa
+
+Check out [this classic 2011 set](https://osu.ppy.sh/beatmapsets/23916) by [Louis Cyphre](https://osu.ppy.sh/users/186243), and [this newer, more modern collaborative set](https://osu.ppy.sh/beatmapsets/1131805) hosted by [Daycore](https://osu.ppy.sh/users/5596337)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/88/previews/2351.mp3" type="audio/mpeg">
+</audio>
+
+### Guitar vs. Piano (ver. XCEED)
+
+Check out [this classic Loved set](https://osu.ppy.sh/beatmapsets/5180036) by [Shiro](https://osu.ppy.sh/users/113005), and [this new collaborative tae](https://osu.ppy.sh/beatmapsets/1195288) hosted by [fieryrage](https://osu.ppy.sh/users/3533958)!
```

Comment:
what's a tae

Reply:
a take where the k ran off to be with its star-crossed lover